schema/json/schema-latest.json

```diff
@@ -3205,6 +3208,33 @@
         "url"
       ]
     },
+    "SnapEntry": {
```

Comment:
@popey - is this the correct naming here `SnapEntry`?
It was SnapMetadata previously. 
Living under the package snap was an anti pattern for how we do the cataloger architecture.

I've lifted it to the correct spot in syft where it's now exported publicly as `pkg.SnapEntry`

If there is a better name I can refactor